src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  width: 100%;
+  height: 100%;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  text-transform: uppercase;
```

Comment:
it's not a good practice to set text-transform: uppercase; as default. 
Use only there where your really need